Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonMessageBox.cs

```diff
@@ -39,7 +39,7 @@ public static class KryptonMessageBox
         /// <param name="contentLinkArea">Specifies the area within the <see cref="KryptonLinkWrapLabel"/> to be regarded as a link. See <see cref="LinkArea"/>.</param>
         /// <param name="messageTextAlignment">Specifies how the message text should be aligned. See <see cref="System.Drawing.ContentAlignment"/> for supported values.</param>
         /// <returns>One of the System.Windows.Forms.DialogResult values.</returns>
-        public static DialogResult Show(string text, string caption, bool? showCtrlCopy = null,
+        public static DialogResult Show(string? text, string caption, bool? showCtrlCopy = null,
```

Comment:
How can the first string be nullable, but the caption not be ?

Reply:
Reverted